2/alpine/Dockerfile

```diff
@@ -37,9 +39,8 @@ RUN set -ex; \
 # need to save initial content for pre-seeding empty volumes
 	mv "$GHOST_CONTENT" "$GHOST_INSTALL/content.orig"; \
 	mkdir -p "$GHOST_CONTENT"; \
-	chown node:node "$GHOST_CONTENT"
```

Comment:
I tried also adding the cleanup lines in this earlier layer and the images weren't quite as small.
```console
ghost               2-alpine-test-layers   d0dc1df5c02c        16 seconds ago      441MB
```

The two layers don't seem like they should be separate anyway.